From 80c647fb2f2dcd6fc45a0d6119c42322a2734bf5 Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Mon, 2 Aug 2021 15:28:31 +0300
Subject: [PATCH 11/19] s32gen1: pcie: Add documentation for PCIE PHY mode

commit 80c647fb2f2dcd6fc45a0d6119c42322a2734bf5 from
https://source.codeaurora.org/external/autobsps32/linux

PCIE PHY mode can be one of: CRNS (default), CRSS or SRIS.

Upstream-Status: Pending

Issue: ALB-6998
Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 Documentation/devicetree/bindings/pci/fsl,s32-pcie.yaml | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/devicetree/bindings/pci/fsl,s32-pcie.yaml b/Documentation/devicetree/bindings/pci/fsl,s32-pcie.yaml
index 1315dda40282..126a67de47e6 100644
--- a/Documentation/devicetree/bindings/pci/fsl,s32-pcie.yaml
+++ b/Documentation/devicetree/bindings/pci/fsl,s32-pcie.yaml
@@ -28,6 +28,14 @@ properties:
     description: A list of interrupt outputs of the controller. Must contain an
       entry for each item in the interrupt-names property.
 
+  fsl,phy-mode:
+    description: Select PHY mode for PCIe controller
+    oneOf:
+      - items:
+          - const: "crns"  # Common Reference Clock, No Spread Spectrum
+          - const: "crss"  # Common Reference Clock, Spread Spectrum
+          - const: "sris"  # Separate Reference Clock, Spread Spectrum
+
   interrupt-names:
     items:
       - const: "link_req_rst_not"
@@ -56,6 +64,7 @@ required:
   - interrupts
   - interrupt-names
   - ranges
+  - nxp,phy-mode
 
 additionalProperties: true
 
-- 
2.17.1

